code/stable/gamephysics/SRS/PDF/GamePhysics_SRS.tex

```diff
@@ -913,7 +913,7 @@ \subsubsection{Data Definitions}
            \end{displaymath}
 \\ \midrule
 Description & \begin{symbDescription}
-              \item{${\symbf{v}\text{(}t\text{)}_{\text{B}}}$ is the velocity at point B ($\frac{\text{m}}{\text{s}}$)}
+              \item{${\symbf{v}\text{(}t\text{)}_{\text{B}}}$ is the Chasles' theorem ($\frac{\text{m}}{\text{s}}$)}
```

Comment:
as noted elsewhere, this is wrong.

Reply:
It looks like we've run into #2788... I'm surprised this issue only appears here because `mkQuantDef'` is used in nearly all of our examples. So why does this issue only appear for `mkQuantDef'`?! It looks like an issue with nesting and inheritance of fields w.r.t. QuantityDicts, QDefinitions, and DataDefinitions.

Reply:
Ok, I believe I fixed the issue. That being said, the type classes around `QDefinition`s+related chunks are all pretty odd. We definitely need to revisit them. I guess our conversation earlier today was right on time!

Reply:
Can you point to the actual use of `mkQuantDef'` that is at fault here?